src/lib.rs

```diff
@@ -286,30 +279,21 @@ impl Default for RequestParams {
 }
 
 /// Get a default `Client` and `RequestParams`.
-pub fn default() -> (hyper::Client, RequestParams) {
-    (hyper::Client::new(), RequestParams::default())
-}
-
-macro_rules! req_with_body {
-    ($client:ident, $url:ident, $req:ident, $params:ident, $method:ident) => ({
-    	let body = $req.body.expect("Expected this request to have a body. This is a bug, please file an issue on GitHub.");
-
-		let body: &[u8] = (*body).borrow();
-		let mut cursor = Cursor::new(body);
-
-		$client.$method(&$url).headers($params.headers.to_owned()).body(&mut cursor).send()
-	})
+pub fn default() -> (reqwest::Client, RequestParams) {
+    //FIXME: Bad in libs
+    let client = reqwest::Client::new().unwrap();
```

Comment:
Looking at this now requiring:

```rust
extern crate elastic_requests;
extern crate elastic_reqwest;
```

It's kinda weird. I think we can just re-export the contents of `elastic_requests` and maybe also the `reqwest::Client`. We can do that in another PR though.

Good point about the `RequestParams`, I've logged an issue #7 for sorting that out.